doc/architecture/decisions/0008-concept-of-bedspaces.md

```diff
@@ -0,0 +1,62 @@
+# 8. concept-of-bedspaces
+
+Date: 2023-02-28
+
+## Status
+
+Accepted
+
+## Context
+
+CAS1 and CAS3 share an API which includes the modelling for premises, rooms and
+beds. This modelling was initially set out by CAS1 and CAS3 have been working to
+extend in order to meet the needs of Temporary Accommodation (CAS3).
+
+The relationships are straight forward. A premises can have many rooms and a
+room can have many beds.
+
+For CAS1 they will often have the scenario of multiple beds belonging to a single
+room.
+
+For CAS3 we only expect one bed per room for the foreseeable future however
```

Comment:
Think this would read better with "foreseeable future however" -> "foreseeable future, however we"